knowledge-base/how-to-display-a-report-via-instancereportsource-in-the-html5-viewer.md

```diff
@@ -0,0 +1,34 @@
+---
+title: How to display a report via InstanceReportSource in the HTML5 Viewer
+description: How to display a report via InstanceReportSource in the HTML5 Viewer.
+type: how-to
+page_title: How to display a report via InstanceReportSource in the HTML5 Viewer
+slug: how-to-display-a-report-via-instancereportsource-in-the-html5-viewer
+res_type: kb
+---
+
```

Comment:
Why are we missing the environment table? This is a mandatory element that helps customer see if the task/issue relates to their environment as if not they will only waste time trying it. More about this can be found here - https://wiki.progress.com/display/DTS/Knowledge+Base+Content+Standards#KnowledgeBaseContentStandards-Environment